src/data/news/software_releases.yml

```
@@ -1,288 +1,222 @@
+- 
+  Title: "Bison Relay Release v0.1.3"
+  Permalink: "https://github.com/companyzero/bisonrelay/releases/tag/v0.1.3"
+  Params:
+    sortDate: "2023-01-20 21:40:16""
```

```suggestion
    sortDate: "2023-01-20 21:40:16"
```

This is causing the build to fail

---

Oops fixed. Good to have the build is checking this.